boards/xtensa/m5stack_core2/doc/index.rst

```diff
@@ -74,6 +74,7 @@ of the M5Stack Core2 board.
 +------------------+--------------------------------------------------------------------------+
 || 3-axis          || The `MPU-6886`_ is a 6-axis MotionTracking device that combines a       |
 || gyrosopce       || 3-axis gyroscope and a 3-axis accelerometer.                            |
```

Comment:
well... now that you mention it ... :D

```suggestion
|| gyroscope       || 3-axis gyroscope and a 3-axis accelerometer.                            |
```

Reply:
I am not an expert on these sensors.I reworked the used terms in both documents (m5stack_core2 and m5stack_core2_ext).